devices/Ft6xx6x/samples/Program.cs

```diff
@@ -45,11 +45,66 @@ void TouchInterrupCallback(object sender, PinValueChangedEventArgs pinValueChang
     {
         var point = sensor.GetPoint(true);
         Debug.WriteLine($"ID: {point.TouchId}, X: {point.X}, Y: {point.Y}, Weight: {point.Weigth}, Misc: {point.Miscelaneous}, Event: {point.Event}");
+        DiscrimitateButtons(point.X, point.Y);
     }
     else if (points == 2)
     {
         var dp = sensor.GetDoublePoints();
         Debug.WriteLine($"ID: {dp.Point1.TouchId}, X: {dp.Point1.X}, Y: {dp.Point1.Y}, Weight: {dp.Point1.Weigth}, Misc: {dp.Point1.Miscelaneous}, Event: {dp.Point1.Event}");
         Debug.WriteLine($"ID: {dp.Point2.TouchId}, X: {dp.Point2.X}, Y: {dp.Point2.Y}, Weight: {dp.Point2.Weigth}, Misc: {dp.Point2.Miscelaneous}, Event: {dp.Point1.Event}");
     }
-}
\ No newline at end of file
+}
+
+void DiscrimitateButtons(int x, int y)
+            {  
+                const double pxPerMm = 6.90415;
+                //r in milimeters is the radious of sensitivity to detect any of the given 3 buttons
+                const double  rMm = 3.5; 
+                const int x0L = 83;
+                const int x0M = 182;
+                const int x0R = 271;
+                const int y0 = 263;
+                const String strLB = "LEFT BUTTON PRESSED";
```

Comment:
Change `String` to lowercase

Reply:
i refactored the whole program, changed String to string but without System the app does not work...